zcash_client_sqlite/src/wallet/init/migrations.rs

```diff
@@ -0,0 +1,24 @@
+mod add_transaction_views;
+
+use schemer_rusqlite::RusqliteMigration;
+use secrecy::SecretVec;
+use zcash_primitives::consensus;
+
+use super::{WalletMigration0, WalletMigration1, WalletMigration2, WalletMigrationError};
+
+pub(super) fn all_migrations<P: consensus::Parameters + 'static>(
+    params: &P,
+    seed: Option<SecretVec<u8>>,
+) -> Vec<Box<dyn RusqliteMigration<Error = WalletMigrationError>>> {
+    vec![
+        Box::new(WalletMigration0 {}),
+        Box::new(WalletMigration1 {}),
+        Box::new(WalletMigration2 {
+            params: params.clone(),
+            seed,
+        }),
+        Box::new(add_transaction_views::Migration {
+            params: params.clone(),
```

Comment:
This feels like it will trigger a clippy lint for "unnecessary clone". I think this should also work:
```suggestion
            params,
```

Reply:
Since we get `params` by reference, we need to clone here. It might be that we should actually make `params` a `Copy` type, but that's not in scope for this PR.